test/iron-overlay-behavior.html

```diff
@@ -165,8 +165,7 @@
         });
 
         test('open() triggers iron-resize', function(done) {
-          // Ignore the iron-resize on attached.
-          var callCount = -1;
+          var callCount = 0;
```

Comment:
So that first `iron-resize` was caused by the node observer which is added on attached. Now that we make sure it is not called during animations, this call count can finally be 0!